third_party/accessibility/ax/ax_node_position_unittest.cc

```diff
@@ -1078,7 +1079,7 @@ TEST_F(AXPositionTest, GetMaxTextOffsetAndGetTextWithGeneratedContent) {
   root_1.role = ax::mojom::Role::kRootWebArea;
   root_1.child_ids = {text_field_2.id};
 
-  text_field_2.role = ax::mojom::Role::kTextField;
+  text_field_2.role = ax::mojom::Role::kGroup;
```

Comment:
Chromium uses kTextField for parent nodes containing further text-containing children. We use it to represent an editable text field; we use kGroup for general grouping nodes.